installer/bundle/bundle_skel_Linux.sh

```diff
@@ -617,14 +602,21 @@ case "$installMode" in
         echo "Updating cross-platform agent ..."
```

Comment:
This may be a good idea, if it can be done and re-tested without too much time lost. However, maybe this should be saved for a separate piece of work.